workspace.spec

```diff
@@ -572,6 +572,378 @@ module Workspace {
 		string description;
 	} ProvenanceAction;
 
+	/* Credit Metadata-related objects and fields
```

Comment:
I'm pretty sure this comment will get dropped by the parser (you could test if it winds up in the html documentation to be sure). If so, you'll want to integrate into a comment for a specific data structure or the module comment

Reply:
It looks like it's integrated into the EventDate comment now, which I'm not sure makes sense - I'd suggest the root data structure in the tree

Reply:
OK, will move it.

Reply:
>		CM is stored as part of the object provenance, but unlike other provenance information, which is immutable, CM can be updated.
>		Credit metadata is added after object creation, and requires the user to be a workspace admin. Username, timestamp, and credit metadata schema version are stored along with the CM, which allows tracking of CM changes.
>		The API does not support updates to select fields; the existing CM for a resource should be fetched, the relevant fields updated, and the resulting structure sent as the input to update credit metadata.

Maybe add this later when it becomes true? I think for the first version it's fixed yeah?

Reply:
Removed until it becomes true

Reply:
:+1: